### DISCUSSION TITLE
Deta Drive file endpoint

### DISCUSSION THREAD

**Discussion by ABSanthosh** [deta/cloud-docs#299] [Help] [answered]
I have a .mp3 file in deta drive at songs/dsf33232dces.mp3. Is there an endpoint to directly get the file instead of using drive.get("dsf33232dces.mp3")?

Thanks in Advance!

**Comment by jnsougata** [comment, marked as answer]
It's not possible at this moment

**Reply by ABSanthosh** [reply]
I guess that's the only way. Thanks for the answer!

**Comment by abdelhai** [comment]
as @jnsougata mentioned, Drive doesn't "serve" files. it is just a service to store and retrieve files. to serve them, you could write your own http server to do so. if you're service small files, you can host your server on a Deta Micro.

**Reply by ABSanthosh** [reply]
I guess that's the only way. Thanks for the answer!
